content.md

```diff
@@ -9,7 +9,7 @@ Our goal is to build an application that allows us to store contact information
 | Carol      | Reynolds  | 20 Oct 2016   | 4556 Mirna Shores   | Apt. 111         | Stromanhaven | DE    | 13654-8312 | 308-571-8066 x3565 | We had a wonderful dinner at La Tavola! |
 | Alice      | Boyer     | 25 Oct 1977   | 7774 Gibson Station | Suite 284        | New Seth     | IN    | 70681      | 984.425.6912       | Ask about their mom                     |
 | Bob        | Stokes    | 30 Nov 1941   | 21817 Lionel Cliffs |                  | South Hailey | CA    | 32418      | +1-487-497-4041    | Loves tulips                            |
-{: .bleed-full }
+{: .table .table-bordered .table-hover }
```

Comment:
By making this table full width instead of scrollable, it is pushed down all the way below the margin ToC: 

---

![image](https://github.com/user-attachments/assets/ce24c36a-9150-4109-8b1f-6afdc3892351)


---

I think it should remain scrollable and not full width. I'm actually surprised it is full width, since I see you dropped it and didn't wrap it in `<div class="bleed-full">`, so I guess `.table` might be causing it to be full width?

Reply:
The table also causes some horizontal overflow when I view the page at 100% zoom.
![learn-table-overflow](https://github.com/user-attachments/assets/3106fa6b-ddcb-4f98-86e4-59ac3fd1fff9)

Reply:
I wrapped it in a div.table-responsive; we should probably do that by default in the Kramdown parser (I thought we already were).

Reply:
LGTM!